framework/src/application/application.ts

```diff
@@ -176,7 +176,9 @@ export class Application {
 
 	public registerPlugin(
 		pluginKlass: typeof BasePlugin,
-		options = {},
+		options: { [key: string]: any; loadAsChildProcess: boolean } = {
```

Comment:
maybe extract this to type.. in future if we expand it might grow and in general we can reuse if required?
`PluginOptions`?